packages/webpack-config/webpack/webpack.common.js

```diff
@@ -436,7 +437,7 @@ module.exports = async function(env = {}, argv) {
         // First, run the linter.
         // It's important to do this before Babel processes the JS.
         // This will throw required errors and warnings during the build.
-        {
+        env.dangerouslyUseLinter && {
```

Comment:
This uses `eslint-config-universe` configuration by default – is it possible to override the configuration using an `.eslintrc` file and have you tested/documented that?

Reply:
will add this in the future, need to disable now so people stop getting #859